src/components/pages/Offerer/OffererCreation/OffererCreationForm/OffererCreationForm.jsx

```diff
@@ -1,32 +1,37 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import Siren from '../Fields/Siren/Siren'
-import Name from '../Fields/Name'
-import Address from '../Fields/Address'
-import PostalCode from '../Fields/PostalCode'
-import City from '../Fields/City'
 import { NavLink } from 'react-router-dom'
+import { FormSpy } from 'react-final-form'
 
 function OffererCreationForm({ handleSubmit, invalid, pristine }) {
```

Comment:
```suggestion
const OffererCreationForm = ({ handleSubmit, invalid, pristine }) => {
```
Préférer l'ES6 pour harmoniser